src/main.rs

```diff
@@ -30,6 +30,7 @@ mod consts;
 mod item;
 mod metadata;
 mod movement;
+mod platform;
```

Comment:
right now this does nothing outside of wasm build correct? Should it be under the same `#[cfg(target_arch = "wasm32")]` that you put the actual code in the plugin under?

Reply:
I could see this being generally useful if we end up needing more platform specific stuff though in which case it would make sense to always leave it in.

Reply:
Yeah, I figured out save data abstraction would go in there once we get to it, so it should always be installed, and we just put our platform specific stuff in there.

Reply:
sounds good